src/Symfony/Bridge/Twig/composer.json

```diff
@@ -22,7 +22,7 @@
     "require-dev": {
         "symfony/asset": "~2.7",
         "symfony/finder": "~2.3",
-        "symfony/form": "~2.7.25|^2.8.18",
+        "symfony/form": "~2.7.26|^2.8.19",
```

Comment:
needed to pull in https://github.com/symfony/symfony/pull/22066/files#diff-e11656489f1e01413320bd5edff0f0c7 to satisfy the `deps=low` job